fiboa_cli/datasets/be_wal.py

```diff
@@ -53,11 +49,14 @@ def layer_filter(self, layer: str, uri: str) -> bool:
     def file_migration(
         self, gdf: gpd.GeoDataFrame, path: str, uri: str, layer: str = None
     ) -> gpd.GeoDataFrame:
-        return gml_assure_columns(
+        gdf = gml_assure_columns(
             gdf,
             path,
             uri,
             layer,
             crop_name={"ElementPath": "specificLandUse@title", "Type": "String", "Width": 255},
             crop_code={"ElementPath": "specificLandUse@href", "Type": "String", "Width": 255},
         )
+        gdf["crop_code"] = gdf["crop_code"].str.extract(r"\.(\d+)$", expand=False)
+        gdf["crop_name"] = gdf["crop_name"].str.strip()
```

Comment:
Why did you move it here?

Reply:
Sounds like we'd need a premigrate and a postmigrate, it feels not ideal to not use the pre-defined hooks for columns etc.